R/XcmsExperiment.R

```diff
@@ -0,0 +1,1707 @@
+#' @title Next Generation `xcms` Result Object
```

Comment:
Cosmetic hint on framing: "next generation" rarely ages well. Similar to "-final" in word files, where inevitably "-final2" crops up.

Reply:
good point - do you have an alternative for a good title?